torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2475,6 +2500,15 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestCommon', 'test_variant_consistency_eager',
                         dtypes=all_types_and_complex_and(torch.bool, torch.bfloat16, torch.float16)),),
            sample_inputs_func=sample_inputs_addr),
+    OpInfo('addcmul',
+           dtypes=all_types_and_complex(),
+           dtypesIfCUDA=all_types_and_complex_and(torch.float16, torch.bfloat16),
+           assert_autodiffed=True,
+           skips=(
+               SkipInfo('TestCommon', 'test_variant_consistency_eager'),
+               SkipInfo('TestGradients', 'test_inplace_grad'),
```

Comment:
I think the TestGradients skips can be replaced with setting supports_inplace_autograd to False:

https://github.com/pytorch/pytorch/blob/0dff0d1537a3746e80dd3a3b655ec071fe3c2466/torch/testing/_internal/common_methods_invocations.py#L166